### ISSUE TITLE
How do we incorporate library resources?

### ISSUE THREAD

**Issue by LASHirsh** [DukeLearningInnovation/learn.duke#79]
(copied from [Questions](https://github.com/DukeLearningInnovation/learn.duke/blob/master/apps/readme.md#questions-on-apps) on /apps)

**Comment by JustinJohnsen** [comment]
What library resources can we incorporate? My assumption is that we are referring to materials stored by and accessed through Duke libraries. Digital media would be easier to conceive, like the way we are automatically able to access many academic journals online through Duke services. That kind of seamless experience would be most desirable I think. Physical resources would be harder. Maybe there are ways to make it even easier for students to know about relevant resources that are available in the library, like scanning an uploaded PDF or text for bibliography references and surfacing their availability at the library in a one-click reservation app.

**Comment by profmikegreene** [comment]
I don’t think we need to answer your question today (and would want to bring in library folks for that conversation anyway) as long as we agree that integrating library systems and resources is on the roadmap and we want to have those conversations in a later sprint. 

My vote is a big yes on this idea, I think robust library system, library resources, and OER integration is something that will make learn.duke feel next gen to users.
